Library/Homebrew/system_command.rb

```diff
@@ -157,8 +163,9 @@ def env_args
 
   sig { returns(T::Array[String]) }
   def sudo_prefix
+    user_flags = sudo_runas_root? ? ["-u", "root"] : []
```

Comment:
```suggestion
    user_flags = []
    user_flags += if ["-u", "root"] sudo_as_root?
```

Reply:
Did you mean `user_flags += ["-u", "root"] if  sudo_as_root?` ? I didn't see this syntax being used in one-liners.

Reply:
@Kentzo yes, sorry!